src/util/apiUtils.test.js

```diff
@@ -254,6 +255,22 @@ describe('parseMetaHeaders', () => {
 	});
 });
 
+describe('parseVariantsHeader', () => {
+	it('parses a variants header into a nested object', () => {
```

Comment:
Possibly worth adding a test case for: "The server may not return a variant for a given experiment, and it may return nothing at all, depending on the availability of the variant service in GCP."

Basically, what happens when the header doesn't return a variant. Also, curious what that looks like... I assume it's `bing-pilot=123|` ?